### PR TITLE
Files from issue #24226

### PR DESCRIPTION
**Pull request by stevecharon**
Hello dregad,

I am sending the files from marnix as of issue #24226 for easy review.
I added an README.md that is linking to Mantis-Wiki-Page for reference. That way it is easier to get on track.

### CODE REVIEW

**Comment by stevecharon** [comment]
Issue https://mantisbt.org/bugs/view.php?id=24226 to not conflict with GH

File: LinkedCustomFields/LinkedCustomFields.php [hunk 1]
```diff
@@ -21,10 +21,10 @@ public function register() {
         $this->name = plugin_lang_get("title");
         $this->description = plugin_lang_get("description");
 
-        $this->version = "1.0.1";
+        $this->version = "2.0.0";
+
         $this->requires = array(
- 			"MantisCore" => "1.2.6",
- 			"jQuery" => "1.8"
+ 			"MantisCore" => "2.0.0, <2.30.0",
```
[the remark this answers is not in the document]
Comment (stevecharon) [reply LinkedCustomFields/LinkedCustomFields.php]: I dont know why. As I mentioned initially, I posted this on behalf of marnix who seems to were unable to do so. But in order to own it:
just drop the max!
```suggestion
 			"MantisCore" => "2.0.0",
```

File: LinkedCustomFields/pages/configure_custom_field_link.php [hunk 2]
```diff
@@ -18,8 +18,14 @@
 	require_once( 'core.php' );
 
 	access_ensure_global_level( config_get( 'manage_custom_fields_threshold' ) );
-	
-	html_page_top( plugin_lang_get( 'configure_custom_field_links' ) );
+
+    html_robots_noindex();
+    header("Content-Security-Policy: default-src 'self'; frame-ancestors 'none'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' img-src 'self'");
```
[the remark this answers is not in the document]
Comment (stevecharon) [reply LinkedCustomFields/pages/configure_custom_field_link.php]: I agree with you. Additionally it doesn't look clean also by itself.
As I don't know enough why it was placed I can't say how to resolve either.

**Comment by eduardosanchez** [comment]
Hi, is this plugin (linked_custom_fields) working on versions 2.0+??

**Comment by dregad** [comment]
I have just updated the _master_ branch, with updated code supporting MantisBT 2.3 or later. Not an official release just yet, testing and feedback would be appreciated.

This PR is therefore obsolete and will be closed.

**Pull request closed without merging**